common/authorization/interceptor.go

```diff
@@ -222,6 +225,9 @@ func (a *Interceptor) Authorize(ctx context.Context, claims *Claims, ct *CallTar
 	if err != nil {
 		metrics.ServiceErrAuthorizeFailedCounter.With(mh).Record(1)
 		a.logger.Error("Authorization error", tag.Error(err))
+		if a.exposeAuthorizerErrors != nil && a.exposeAuthorizerErrors() {
```

Comment:
This should never be `nil` right?

Reply:
I would rather us fail fast and crash than have a situation where the flag is not properly passed to the authorizer.